dependencyVersions.gradle

```diff
@@ -23,14 +23,17 @@ project.ext.groupIdOkio = 'com.squareup.okio'
 project.ext.artifactIdOkio = 'okio'
 
 // Kotlin
-project.ext.versionKotlinScriptRuntime = '1.3.0'
-project.ext.groupIdKotlinScriptRuntime = 'org.jetbrains.kotlin'
+project.ext.versionKotlin = '1.3.41'
+project.ext.groupIdKotlin = 'org.jetbrains.kotlin'
 project.ext.artifactIdKotlinScriptRuntime = 'kotlin-script-runtime'
+project.ext.artifactIdKotlinMainKts = 'kotlin-main-kts'
 ext.kotlin = [
-        runtime : "$groupIdKotlinScriptRuntime:$artifactIdKotlinScriptRuntime:$versionKotlinScriptRuntime"
+        runtime : "$groupIdKotlin:$artifactIdKotlinScriptRuntime:$versionKotlin" ,
+        mainKts: "$groupIdKotlin:$artifactIdKotlinMainKts:$versionKotlin"
```

Comment:
Do you need this anymore?

Reply:
actually i would like to keep it at the moment, is not used i know, but maybe soon we can find the way to remove the curl hack.. and in this case we need this

Reply:
Mmmm we will add it back then I suppose.
Because is we manage to fix it in a different way, we could forget of this, in case we need it will still be in the git history